junifer/data/parcellations.py

```diff
@@ -700,3 +700,84 @@ def _retrieve_suit(
     ].to_list()
 
     return parcellation_fname, labels
+
+
+def merge_parcellations(
+    parcellations_list: List,
```

Comment:
Is it `List[str]`?

Reply:
no, it should be `List["Nifti1Image"]` i think, should i put it?

Reply:
Yeah